sphinx/source/docs/user_guide/compat.rst

```diff
@@ -43,29 +39,18 @@ in a web browser.
     :alt: Datashader Bokeh example
     :align: center
 
-Datashader works well together with `HoloViews`_ (see below and
-`this_example`_), which allows you to flexibly switch between
-datashaded and non-datashaded versions of a plot, interleave
-datashader-based plots with other Bokeh-based plots, and so on.
-
-
 HoloViews
 ---------
 
-Bokeh is designed to provide an enormous amount of power and
-flexibility to the Python programmer, making it feasible to develop
-complex visualization-focused applications for deployment on web
-browsers. However, for day-to-day work exploring and visualizing
-data, it can be helpful to have a higher-level API on top of what
-Bokeh provides, to make it simpler to do common visualization tasks
-without specifying each step explicitly.
-
-`HoloViews`_ is a separately maintained package that provides a
-concise declarative interface for building Bokeh plots. HoloViews is
-particularly focused on interactive use in a Jupyter notebook,
-allowing quick prototyping of figures for data analysis. For
-instance, to build an interactive figure with three linked Bokeh plots
-requires only one line of code in HoloViews:
+Bokeh offers you a lot of versatility when it comes to development
```

Comment:
Should'nt this be either "when it comes to the development of complex" or "when it comes to developing complex"?

Reply:
Seems ok to me already, but I probably wrote it, so it would! :-) Either revision sounds fine.

Reply:
I'd prefer "when it comes to developing complex" - but up to @Konntroll, I'd say 😀